app/src/main/java/me/quadphase/qpdex/PokedexActivity.java

```diff
@@ -267,7 +295,7 @@ public void switchToPokemonData(View view){
         startActivity(intent);
     }
 
-    public void playPokemonCry(){
+    public void playPokemonCry(View view){
```

Comment:
When you create an OnClick method, it asks for a View parameter if you want to link it from the XML. I left that possibility open here for now...